src/features/profile/ProfilePage.tsx

```diff
@@ -39,6 +40,7 @@ const ProfilePage = ({ customerContactInfo, hasBerthNotifications, hasWSNotifica
                   {t('page.profile.berths.title')}
                 </span>
               </Tab>
+              <Tab>BerthOffer</Tab>
```

Comment:
I assume this is a temporary tab for testing, right?

Reply:
Yes, as we currently have no way to otherwise test these different views. I wrote about this in the description.

Food for thought, how will we actually test these in the future? We can't impersonate other users